core/src/consensus.rs

```diff
@@ -347,6 +345,22 @@ impl Tower {
         last_vote
     }
 
+    fn maybe_timestamp(&mut self, current_slot: Slot) -> Option<UnixTimestamp> {
```

Comment:
nits: no longer maybe?

Reply:
I considered removing the `maybe_` (even did it locally once) but the function does still return an Option because the function will not add a timestamp to a slot that's older than the latest slot that has been timestamped. 
(although this behaviour might be unnecessary once tower save/restore is working...)